docs/dev_docs/verifier.md

```diff
@@ -0,0 +1,10 @@
+# CSV Verifier
+The merger and fetcher use a CSV verifier to ensure that exact duplicates don't occur as well as to ensure the integrity of the CSV file (e.g. does it have commas as separators). The verifier is located in the Verifier package and can take any amount of filepaths. If the verifier finds a problem it will return an error. 
+```go
+func Verifier (filepaths ... string) error 
+```
+It checks the documents line by line and adds blips to a set of seen names and will remove any duplicate found in the future if it is in the same ring (this may be changed for LLM duplicate handling later). 
+
+The verifier uses a alt_names map with a dictionary file (will come later) to ensure that alternative names are counted as the same thing. E.g. C#, CSharp, CS will all be mapped the same value ensuring they are counted as the same thing.
+
+*Future expansion of will allow for header verification as well*
```

Comment:
I'm unsure if we should be writing how the codebase will work in the future rather than how it works now, i.e. line 8. I'd suggest keeping all future extensions seperate from current implementation explanations as seen below.

```suggestion
*Future extensions include...*
- Dictionary file
  - The verifier will use an `alt_names map` with a dictionary file to ensure that alternative names are unified. E.g. C#, CSharp, CS will all be mapped the same value, ensuring they are processed as equivalent.
- Allowing for header verification
```

Reply:
Problem is it does have a basic alt names map just not tied to a file so I agree with removing the future stuff but we need to rewrite it to have the actual functionality instead then

Reply:
I have changed the docs please look at it